NextcloudTalk/NCRoomsManager.m

```diff
@@ -804,8 +804,15 @@ - (void)joinOrCreateChatWithURL:(NSNotification *)notification
 {    
     NSString *accountId = [notification.userInfo objectForKey:@"accountId"];
     NSString *withUser = [notification.userInfo objectForKey:@"withUser"];
+    NSString *roomToken = [notification.userInfo objectForKey:@"withRoomToken"];
     [self checkForAccountChange:accountId];
-    [self joinOrCreateChatWithUser:withUser usingAccountId:accountId];
+
+    if (withUser != nil) {
+        [self joinOrCreateChatWithUser:withUser usingAccountId:accountId];
+    } else if (roomToken != nil) {
```

Comment:
We could check `withUser` and `roomToken` as

```
if (withUser) {
    [self joinOrCreateChatWithUser:withUser usingAccountId:accountId];
} else if (roomToken) {
```
same as you did in NCUserInterfaceController.m ;)